.travis.yml

```diff
@@ -55,8 +55,5 @@ script:
    - travis_wait 30 ./gradlew :libs:SalesforceHybrid:connectedAndroidTest
    - ./gradlew :libs:SalesforceReact:assembleDebug
    - travis_wait 30 ./gradlew :native:NativeSampleApps:RestExplorer:connectedAndroidTest
-   # make sure the screen is unlocked
-   - adb shell input keyevent 82
-   - adb shell input keyevent 84
-   - adb shell input keyevent 84
-   - travis_wait 30 ./gradlew :native:TemplateApp:connectedAndroidTest
+   # disable this ui test until fix the unstable test failure 
+   - ./gradlew :native:TemplateApp:assembleDebug
```

Comment:
That sounds good, but we have disabled and re-enabled this specific test many times. If I'm not mistaken, it's not a legitimate failure, it's a flapper. Also, it appears that Travis CI itself is not stable - tests time out a lot. Given this, I don't think we should disable or re-enable until we make the environment (Travis CI) stable and make the flapping tests stop flapping.